mos_tests/failover/conftest.py

```diff
@@ -0,0 +1,54 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+
+import pytest
+from waiting import wait
+
+from mos_tests.environment.os_actions import OpenStackActions
+
+
+logger = logging.getLogger(__name__)
+
+
+def is_ha(env):
```

Comment:
Not really. Also `check_env_` will not work here (because it logic realized in https://github.com/Mirantis/mos-integration-tests/blob/master/mos_tests/neutron/conftest.py#L92)